sbncode/DetSim/fcl/icarus_simedepfilter.fcl

```diff
@@ -0,0 +1,43 @@
+BEGIN_PROLOG
+
+simedepfilter_ind1gap: {
+  module_type: "FilterSimEnergyDeposits"
+  InitSimEnergyDepositLabel: "ionization"
+  OutSimEnergyDepositLabel: "filteredSED"
+  P1_X: -400
+  P1_Y: -200
+  P1_Z: -2
+  P2_X: 400
+  P2_Y: 200
+  P2_Z: 2
+}
+
+simedepfilter_cathodewest: {
+  module_type: "FilterSimEnergyDeposits"
+  InitSimEnergyDepositLabel: "ionization"
+  OutSimEnergyDepositLabel: "filteredSED"
+  P1_X: 192.2
+  P1_Y: -200
+  P1_Z:	-900
+  P2_X: 232.2
+  P2_Y:	200
+  P2_Z:	900
+}
+
+simedepfilter_cathodeeast: {
+  module_type: "FilterSimEnergyDeposits"
+  InitSimEnergyDepositLabel: "ionization"
+  OutSimEnergyDepositLabel: "filteredSED"
+  P1_X:	-232.2
+  P1_Y: -200
+  P1_Z: -900
+  P2_X:	-198.2
+  P2_Y: 200
+  P2_Z:	900
+}
+
+filter_ind1gap: @local::simedepfilter_ind1gap
+filter_cathodewest: @local::simedepfilter_cathodewest
+filter_cathodeeast: @local::simedepfilter_cathodeeast
```

Comment:
What is the idea behind these aliases?

Reply:
Mostly ignorance, don't need these can use the original names in the same file, removed